### ISSUE TITLE
Change Monthly Meeting?

### ISSUE THREAD

**Issue by mrocklin** [dask/community#376]
Attendance has been low for a while and conversations have been lacking.  I think that we're mostly handling the conversations here asynchronously.  

Still very much in favor of talking, but maybe not regularly.

**Comment by jacobtomlinson** [comment]
We used to open an issue here ahead of each community meeting, I noticed attendance has slumped a lot we stopped doing that. I wonder if it's an advertising problem rather than a demand problem?

**Comment by mrocklin** [comment]
Even when more people showed up I'm not sure that the meeting achieved much.  

Maybe regardless of whether or not we keep the meeting we should step back and ask what we're trying to achieve with it, and then ask what the best ways to achieve those things are (maybe the meeting, maybe something else).

**Comment by hendrikmakait** [comment]
> Maybe regardless of whether or not we keep the meeting we should step back and ask what we're trying to achieve with it, and then ask what the best ways to achieve those things are (maybe the meeting, maybe something else).

+1 on this. 

While I think that advertising could definitely be improved, the bigger problem I see is that the content of the Dask Monthly Meeting overlaps significantly with the Dask Demo Days and the Maintainers Sync. Updates on the progress of `dask` or `dask-contrib` projects have historically been a big part of the monthly. Guess what? Demos would be a great way to highlight the recent work that has been going on. Wherever that doesn't apply for some reason (this might also be due to a lack of time by the developers), we could reserve a small section of the demo days for development announcements to the wider Dask community. Another part has been synchronization among core developers and discussions around releases. I feel like we have recently shifted toward this in the Maintainers Sync, so there's probably less need for that as well. As for new member proposals, it might be nice to give them a glimpse at what we do as maintainers, so I'd be happy for them to "visit" the maintainers sync for this. (Technically, that's not even necessary.)

There's probably some important part of the monthly that doesn't fit as naturally into one or the other meeting that I have overlooked, so please let me know if anything is missing.

**Comment by mrocklin** [comment]
I like the idea of leaning on demo day for the community aspect that the monthly meeting was kinda providing

**Comment by jacobtomlinson** [comment]
Yeah I like that idea too. I think having a place where folks show what is going on in the Dask community is something I don't want us to lose. But maybe merging the demo day and the community call into a single monthly activity is the way to do it.

**Comment by jacobtomlinson** [comment]
How about this as a proposal:

The next community meeting is on June 6th. Let's cancel the Demo Day (#380) and also the maintainers sync and fold everything into the community meeting. I also think sharing updates and giving demos should be the primary agenda item, followed by maintainer discussion and more niche chat, then wrapping up with some social time.

We should get back into the habit of opening an issue here a few days before the meeting and also promoting it on Twitter.

**Comment by mrocklin** [comment]
I don't object to the proposal.  @jacobtomlinson would you be ok organizing the next few meetings like this?  Maybe transferring that responsibility to you gives you more of a sense of ownership and encourages you to play around and think about solutions.  It also reduces expectations of effort on my end.

**Comment by jacobtomlinson** [comment]
Happy to!

**Comment by charlesbluca** [comment]
Would it make sense to switch the "introductions" and "opens" sections of the combined meeting format? Feel like this might be a better segue from some of the technical discussions prompted by the preceding demos section.
